### PR TITLE
In App Guidence

### PR DESCRIPTION
**Pull request by schandlergarcia**
### What does this PR do?

This update introduces In-App Guidance to E-bikes. 

This includes walkthroughs for the: 

- Product Explorer
- Order Builder
- Product Detail Page

### What issues does this PR fix or reference?

None

## The PR fulfills these requirements:

[X] Tests for the proposed changes have been added/updated.
[X] Code linting and formatting was performed.

### Functionality Before

There was no in-app guidance :( 

### Functionality After

We now have in-app guidance :)

### CODE REVIEW

File: force-app/main/default/prompts/OrderBuilder.prompt-meta.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        <targetPageKey2>view</targetPageKey2>
+        <targetPageType>standard__recordPage</targetPageType>
+        <title>Order Details</title>
+        <userAccess>Everyone</userAccess>
+        <userProfileAccess>Everyone</userProfileAccess>
+        <versionNumber>1</versionNumber>
+    </promptVersions>
+    <promptVersions>
+        <body
+        >&lt;p&gt;Now that you have seen Lightning Web Components used to create an interactive ordering experience, you can learn how to build them yourself! &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;As demonstrated, you can follow web standards like using Custom Events &amp;amp; The Drag &amp;amp; Drop API to communicate across components. &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Make sure you check out the &lt;b&gt;Property Explorer &lt;/b&gt;Tab for more great ways you can innovate with Lightning Web Components.&lt;/p&gt;</body>
```
Comment (pozil) [inline force-app/main/default/prompts/OrderBuilder.prompt-meta.xml:100]: This is calling out the "Property Explorer", I guess that you mean "Product Explorer"

File: force-app/main/default/prompts/OrderBuilder.prompt-meta.xml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        <targetPageKey2>view</targetPageKey2>
+        <targetPageType>standard__recordPage</targetPageType>
+        <title>Product List</title>
+        <userAccess>Everyone</userAccess>
+        <userProfileAccess>Everyone</userProfileAccess>
+        <versionNumber>1</versionNumber>
+    </promptVersions>
+    <promptVersions>
+        <body
+        >You can drag products from the list into the basket. As you add products, the totals will update and relate the records to the order. &lt;br&gt;&lt;br&gt; The drop is handled using the standard &lt;a href=&quot;#=https://mzl.la/36N7i4c&quot;&gt;Drag &amp; Drop APIs&lt;/a&gt;.</body>
```
Comment (pozil) [inline force-app/main/default/prompts/OrderBuilder.prompt-meta.xml:60]: Something's wrong with the link here. I can't use it, it closes the dialog box.

File: force-app/main/default/prompts/ProductExplorer.prompt-meta.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        <targetPageKey1>Product_Explorer</targetPageKey1>
+        <targetPageType>standard__navItemPage</targetPageType>
+        <title>Filters</title>
+        <userAccess>Everyone</userAccess>
+        <userProfileAccess>Everyone</userProfileAccess>
+        <versionNumber>1</versionNumber>
+    </promptVersions>
+    <promptVersions>
+        <body
+        >E-Bikes products are show in this paginated list.&lt;br/&gt;Click on one of the tiles to view product details.</body>
```
Comment (pozil) [inline force-app/main/default/prompts/ProductExplorer.prompt-meta.xml:48]: I realized that I made a typo here: missing "n" in "shown"

**Review by pozil — CHANGES_REQUESTED** [review]
Found some minor issues.

File: force-app/main/default/prompts/OrderBuilder.prompt-meta.xml (hunk 2 above)
Comment (schandlergarcia) [reply force-app/main/default/prompts/OrderBuilder.prompt-meta.xml]: I've removed this as it doesn't seem we can hyperlink outside of salesforce.

**Comment by schandlergarcia** [comment]
@pozil I've made the requested updates to the prompts, let me know if we're ok to merge.

**Review by pozil — APPROVED** [review]
All good!

**Pull request merged**